data_model/1.3/clusters/BallastConfiguration.xml

```diff
@@ -118,37 +152,46 @@ Davis, CA 95616, USA
     </attribute>
     <attribute id="0x0014" name="IntrinsicBallastFactor" type="uint8">
       <access read="true" write="true" readPrivilege="view" writePrivilege="manage"/>
-      <quality changeOmitted="false" nullable="true" scene="false" persistence="volatile" reportable="false" sourceAttribution="false" quieterReporting="false"/>
+      <quality nullable="true"/>
       <optionalConform/>
     </attribute>
     <attribute id="0x0015" name="BallastFactorAdjustment" type="uint8" default="null">
       <access read="true" write="true" readPrivilege="view" writePrivilege="manage"/>
-      <quality changeOmitted="false" nullable="true" scene="false" persistence="volatile" reportable="false" sourceAttribution="false" quieterReporting="false"/>
+      <quality nullable="true"/>
       <optionalConform/>
-      <constraint type="between" from="100" to="MS"/>
+      <constraint>
+        <between>
+          <from value="100"/>
+          <to value="MS"/>
+        </between>
+      </constraint>
     </attribute>
     <attribute id="0x0020" name="LampQuantity" type="uint8">
       <access read="true" readPrivilege="view"/>
       <mandatoryConform/>
     </attribute>
-    <attribute id="0x0030" name="LampType" type="string" default="&quot;">
+    <attribute id="0x0030" name="LampType" type="string" default="&quot;&quot;">
```

Comment:
Should default for empty strings be just an emtpy string rather than the string `""` ?

Reply:
asked the question here: https://github.com/project-chip/alchemy/issues/23

But again, this is just reflective of the lack of consistency in the spec.